product_docs/docs/sql_assess/1/troubleshooting.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Troubleshooting EDB SQL Assess
+navTitle: Troubleshooting
+---
+
+The EDB SQL Assess tool generates a log file in the same path where the report is generated. The path is provided by the user in the command line options. The tool supports three log levels. If set to `OFF` then log is not generated. If set to `INFO`, which is the default, then all messages are logged except the queries that are assessed. If the level is set to `FINE` then the queries that are assessed are also logged in the file.
+
+To generate reports correctly, EDB SQL Assess needs to resolve the `hostname` to a valid IP address. The easiest way is to add a line like the following in the `/etc/hosts` file:
+
+```
+127.0.0.1	myhost
+```
+
+If the `hostname` can't be resolved to a valid IP address, EDB SQL Assess might report an error like the following:
+
+```
+myhost: Name or service not known
+```
+
+You can run EDB SQL Assess from a folder where the user has permission to create files and folders, otherwise an error like the following is reported:
+
+```
+Cannot create directory 'plte_cache/c75542c1-4763-4ead-a06c-c3534ed2687b'.
+```
+
+!!! Note
```

Comment:
This doesn't leave a lot of confidence in the extractor.

@gabbasb can you provide a bit more context around conditions where not all queries are extracted?  Is it the case, that the queries aren't available to be extracted because they are not in the view or that there is some bug causes the extractor not to be able output some queries in the view?

Reply:
@mlewandowski-edb there is no known bug in the extractor causing this behaviour. However because of some unknown issue some queries (mostly DDLs) are not there in the V$SQL view and are therefore not extracted by the extractor script. We had tried but were not able to find the exact reason.

Reply:
This is not an issue with the extractor.  Rather, the issue has to do with Oracle not making certain queries available to the V$SQL view.  I will update the text to reflect that.